File: A_Gravity_Flip.java
import java.util.Scanner;

class A_Gravity_Flip{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        sc.nextLine();
        int[] arr = new int[n];

        for(int i = 0; i < n; i++){
            arr[i] = sc.nextInt();
        }

        for (int i = 0; i < n; i++){
            for(int j = 0; j < n - 1; j++){
                if(arr[j] > arr[j + 1]){
                    int dif = arr[j] - arr[j + 1];

                    arr[j + 1] += dif;

                    arr[j] -= dif;
                }
            }
        }

        for(int i : arr) System.out.print(i + " ");
    }
}

File: A_Gravity_Flip.py
n = int(input())

arr = list(map(int, input().split()))


for i in range(n - 1):
    if arr[i] > arr[i + 1]:
        dif = arr[i] - arr[i + 1]
        arr[i + 1] += dif
        arr[i] -= dif
    
print(*arr)